### DISCUSSION TITLE
从ocr-github官方下载ocr-release-2.7，打开train.py文件之后内部报错但不影响运行

### DISCUSSION THREAD

**Discussion by sssjc666** [PaddlePaddle/PaddleOCR#12544] [Q&A]
打开文件之后发现内部代码报错，如果对ppocr进行../ppocr修改会导致代码无法正常运行，不进行修改代码可以运行但是我无法用ctrl+函数名跳转至另一个函数内部，各位有遇到这个问题的吗？请问是如何解决的？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/126962940/74512c2c-653e-4218-89e5-45256aa6e0a3)

**Comment by SWHL** [comment]
如问题仍然存在，请拉取最新main分支代码再次尝试。
